src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,67 @@
+package com.wolox.training.controllers;
+
+import com.wolox.training.dtos.BookDTO;
+import com.wolox.training.models.Book;
+import com.wolox.training.services.BookService;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+
+    @Autowired
+    private BookService bookService;
+
+    @GetMapping
+    public ResponseEntity<List<Book>> findAll() {
+        List<Book> books = bookService.findAll();
+        return new ResponseEntity<>(books, HttpStatus.OK);
+    }
+
+    @GetMapping("/book/{bookAuthor}")
```

Comment:
This is not very RESTful, to be honest. At most you could change the word book for "author", but it's not really the best way to filter by attributes.
But don't worry, you actually don't need to add this endpoint for this Card, only the other ones you did.
Later in the training you'll do this filtering with query parameters.
For the sake of this card (And given you already did some work on it), just change `book` for `author` and that's it.